engine/src/exec.cpp

```diff
@@ -1573,14 +1573,32 @@ static bool MCPropertyParseUIntList(MCStringRef p_input, char_t p_delimiter, uin
 	uindex_t t_new_offset;
 	t_new_offset = 0;
 	
+	// PM-2015-10-13: [[ Bug 16203 ]] Replace any "empty" elements with 0 in the beginning of the list
+	if (MCStringGetNativeCharAtIndex(p_input, 0) == p_delimiter)
```

Comment:
I'm not sure this clause is needed if you remove the 't_new_offset <= t_old_offset' check from the while condition